src/pages/terms/ci.md

```diff
@@ -0,0 +1,367 @@
+  regulation or rule, or the rights of any third party including without
+  limitation Intellectual Property Rights;
+- Post or transmit viruses, Trojan horses, worms, spyware, time bombs,
+  cancelbots, or other computer programming routines that may harm the Services
+  or interests or rights of other users, or that may harvest or collect any data
+  or personally identifiable information about other users without their
+  consent;
+- Post or transmit any information, data, Application(s) or User Data that is
+  unlawful, harmful, abusive, racially or ethnically offensive, defamatory,
+  infringing, invasive of personal privacy or publicity rights, harassing,
+  humiliating to other people (publicly or otherwise), libelous, threatening, or
+  otherwise objectionable;
+- Use or launch any automated system, including without limitation, “robots,”
+  “spiders,” “offline readers,” etc., that accesses the Services in a manner
+  that sends more request messages to the Shorebird servers than a human can
+  reasonably produce in the same period of time by using a conventional on-line
+  web browser;
+- Attempt to gain unauthorized access to any other party’s Account, password,
+  Application or User Data, or allow more than one person to use an Account; or
+- Use the Services in any manner that violates applicable law or infringes the
+  intellectual property or other rights of any third party.
+
+## F. License Grant to Us of Your Source Code/Application and User Data In Order for Us to Provide You the Services; Restriction.
+
+You hereby grant to Us, solely during the applicable term of Your use of the
+Services, a limited, personal, non-exclusive, worldwide, license to access,
+retrieve, use, and temporarily store the source code to Your Application and
+User Data solely to the extent necessary to perform the Services under this
+Agreement. This includes the right to: (a) access the repositories linked to the
+Services through the GitHub Marketplace or similar integration; (b) detect and
+respond to pull requests initiated by You; (c) execute tests, validations, and
+verifications on code submitted in such pull requests; and (d) generate and
+deliver the results (including test outcomes, logs, and analysis) to You through
+GitHub or other designated channels.
+
+Except as necessary to provide the Services, we will not: (a) modify,
+distribute, or create derivative works of the source code to Your Application;
+(c) retain or store the source code to Your Application; (d) use the source code
+to the Your for machine learning or artificial intelligence model training; or
+(e) disclose the source code to Your Application to third parties, except to
+sub-processors or infrastructure providers subject to appropriate
+confidentiality obligations.
+
+## G. Fees and Billing
+
+We provide the Services for the fees, if any, and other charges set forth in our
+pricing section here https://shorebird.dev/pricing. All prices listed exclude
```

Comment:
Not sure what we want to do here.

Reply:
I think we can leave it since eventually we'll probably want all of our plans/pricing visible from `/pricing` 🤷